a3p-integration/proposals/c:price-feeds/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "agoricProposal": {
+    "releaseNotes": "TBD",
+    "sdkImageTag": "unreleased",
+    "planName": "UNRELEASED_UPGRADE",
+    "upgradeInfo": {
+      "coreProposals": []
```

Comment:
if `coreProposals` is empty, what exactly is happening in this Software Upgrade?

Reply:
It runs the upgrade handlers in the `sdkImageTag`. But that will have run by `a:upgrade-next`, so its's not doing anything. 

It looks like this proposal package is just a holder for `priceFeed.test.js` (and its submission).